.travis.yml

```diff
@@ -3,9 +3,8 @@ matrix:
   include:
     - language: go
       sudo: false
-      go: "1.10"
+      go: "1.13"
```

Comment:
We need to confirm that the base container image with which we ship (i.e. UBI) supports 1.13.

Reply:
Go modules full support introduced from go 1.12 on-words.

Reply:
I am not referring to gomod, I am referring to the golang version 1.13, and verifying it being available in the UBI base container version that we intend to use. For instance, RHEL 7 did not have golang 1 .13.